src/font/lv_font_fmt_txt.c

```diff
@@ -86,16 +86,19 @@ const void * lv_font_get_bitmap_fmt_txt(lv_font_glyph_dsc_t * g_dsc, lv_draw_buf
 
     const lv_font_fmt_txt_glyph_dsc_t * gdsc = &fdsc->glyph_dsc[gid];
 
+    if(g_dsc->req_raw_bitmap) return &fdsc->glyph_bitmap[gdsc->bitmap_index];
```

Comment:
I didn't check too deeply but is this ok to be returning a `uint8_t *` while normally from this function a `lv_draw_buf_t *` is returned?